hcp-consul-hcp-vault-ca/modules/kubernetes/main.tf

```
@@ -43,6 +43,19 @@ resource "kubernetes_config_map" "aws_profile_config" {
   }
 }
 
+resource "kubernetes_config_map" "consul_values" {
+  metadata {
+    name = "consul-values.yaml"
```

```suggestion
    name = "consul-values"
```
Name is not valid.  TF is throwing an error when running this.